Game/Src/Powers/LightningPower.gd

```diff
@@ -0,0 +1,7 @@
+extends "res://Src/Powers/Powers.gd"
+
+export var texture = preload("res://Assets/UI/LightningPwr.png")
+
+func lightning(closest_enemy):
```

Comment:
maybe it would be easier to call all functions "apply_power" but keep it defined as it is now in each specific power.
This way, in the future we can always write power.apply_power() whatever the power. And we can potentially use it in scripts

Reply:
Agreed, I was thinking of something similar but couldn't wrap my mind around it at the time so I went for the easy fix, focusing on 'nodification' first